tests/acceptance/cli/ursula/test_ursula_command.py

```diff
@@ -39,7 +39,8 @@ def capture_output():
 @pytest.fixture(scope='module')
 def ursula(federated_ursulas):
     ursula = federated_ursulas.pop()
-    return ursula
+    yield ursula
+    ursula.disenchant()
```

Comment:
Should we do `ursula.stop()` here instead?

Reply:
Hmm - Somewhat of an open question re: combining `stop` and `disenchant` ( @jMyles )